### DISCUSSION TITLE
Версионность файлов

### DISCUSSION THREAD

**Discussion by Makhmut2445** [totumonline/totum-ru-issues-and-discussions#473] [Как это сделать?] [answered]
Как включить версионность файлов не включая защиту?
![image](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/103171127/b60e4236-2e97-4522-bda4-c20c690a0430)

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Никак

**Reply by adu05** [reply]
Подскажите пожалуйста, каким образом в этом случае выполнить копирование файла, либо создание ссылки на файл, в соответствии с задачей бизнес-процесса в другую сущность (таблицу)

**Reply by ttm-alexeypolunin** [reply]
Ссылку нельзя. Возможно когда-нибудь будет. Копировать можно - fileGetContent получит содержимое последней версии.

**Reply by ttm-alexeypolunin** [reply]
linkToEdit мне кажется самым оптимальным для таких случаев
